docs/template-vm.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: linux-vm-template
```

Comment:
Can we use a specific name, like rhel-7-template

Then it will be clearer further below when we reference disk images.

Reply:
I though about naming it rhel-7, but no settings was rhel-7 specific, so I choose to be more generic. I can switch to rhel-7 if reviewers prefer.

Reply:
For the time being still kept linux-vm-template, will do another upload switching to rhel-7 if needed later

Reply:
wfm

Reply:
Is parameter describing OS details?

Reply:
No. The field (name) is just defining the template name - it is technically not tied to functionality, just an identifier.

Reply:
so how are we going to identify OS version?

Reply:
Some time ago we had the discussion which metadata was needed. Name, CPU, memory, flag for identifing applicable templates were the ones named.

It sounds as if you also need to understand which OS is used by a template, is this correct? Do you need a way to extract this information?